packages/botonic-plugin-flow-builder/src/content-fields/flow-element.tsx

```diff
@@ -36,7 +36,7 @@ export class FlowElement extends ContentFieldsBase {
         <Pic src={this.image} />
         <Title>{this.title}</Title>
         <Subtitle>{this.subtitle}</Subtitle>
-        {this.button?.renderButton()}
+        {this.button?.renderButton(0)}
```

Comment:
If you have more than one element inside a Carousel and you always pass the number 0 as buttonIndex, if two elements point to the same target they will be the same. Maybe this buttonIndex should come from the element number inside the Carousel. 
I know this can't happen in Whatsapp but the buttonIndex should be different.

Reply:
by now we only support on button per element in the flow builder, if we add the option of having more button we'll change it